.github/workflows/main.yml

```diff
@@ -30,6 +30,11 @@ jobs:
         run: |
           python kapew.py prep-kolibri-dist --exclude-prereleases --skip-preseed
 
+      - name: Apply performance patches
+        run: |
+          cp patches/morango-models-utils.py src/kolibri/dist/morango/models/utils.py
+          cp patches/ifcfg-parser.py src/kolibri/dist/ifcfg/parser.py
```

Comment:
If we're applying the `dist/morango/models/utils.py` patch, we don't need to touch `dist/ifcfg/parser.py`. (The _ifcfg_ change removes a slow call to `socket.gethostbyaddr`, while the  _morango_ change monkey patches `socket.gethostbyaddr` to return a constant instead).

Reply:
Ok, I've just removed the second patch because it's not needed.